### PR TITLE
Add accessibiltyDescription prop to View config 

### PR DESCRIPTION
**Pull request by lenahong**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/6970)

### CODE REVIEW

File: packages/react-native-win32/src/Libraries/Components/View/ReactNativeViewViewConfig.win32.js [hunk 1]
```diff
@@ -381,6 +381,7 @@ const ReactNativeViewConfig: ViewConfig = {
     cursor: true,
     accessibilityDescribedBy: true,
     accessibilityLabeledBy: true,
+    accessibilityDescribedBy: true,
```
Comment (NikoAri) [inline packages/react-native-win32/src/Libraries/Components/View/ReactNativeViewViewConfig.win32.js]: >accessibilityDescribedBy: true, [](start = 4, length = 31)

Isn't this a duplicate entry from line 382?

**Review by NikoAri — CHANGES_REQUESTED** [review]
:clock1:

File: packages/react-native-win32/src/Libraries/Components/View/ReactNativeViewViewConfig.win32.js (hunk 1 above)
Comment (lenahong) [reply packages/react-native-win32/src/Libraries/Components/View/ReactNativeViewViewConfig.win32.js]: fixed in third commit!

**Review by NickGerleman — APPROVED** [review]
Need to run "yarn lint:fix"

**Review by FalseLobster — APPROVED** [review]

**Review by NikoAri — APPROVED** [review]
:shipit:

**Pull request merged**